docs/index.rst

```diff
@@ -21,7 +21,7 @@ deeper into the system :ref:`reference`.
 For technical documentation on the Superdesk client web application, have a look
 `here <https://github.com/superdesk/superdesk-client-core>`_.
 
-.. admonition:: Looking for non-technical user documentation?
+.. admonition: Looking for non-technical user documentation?
```

Comment:
this is actually part of the markup, you can try `.. hint::` or `.. tip::` instead